python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -33,10 +33,19 @@
 from paddle.fluid.op import Operator
 from paddle.fluid.executor import Executor
 from paddle.fluid.framework import Program, OpProtoHolder, Variable
-from testsuite import create_op, set_input, append_input_output, append_loss_ops
+from paddle.fluid.tests.unittests.testsuite import (
```

Comment:
why change this file?

Reply:
Since this is the recommended Python style for importing multiple modules from single package [PEP328](https://www.python.org/dev/peps/pep-0328/#id7) and generally it looks more readable.